### ISSUE TITLE
Accessibility: Support markdown syntax for strikethrough for the text widget

### ISSUE THREAD

**Issue by caaster** [projectblacklight/spotlight#3484]
### Issue Description - WCAG standard
When images on a home, feature, or about page contain text, that text must be transcribed in full to comply with accessibility requirements -- typically using a feature page titled "Appendix". Sometimes the text to be transcribed is a manuscript or typewritten page that contains cross-outs. We need a way to indicate this in the transcript, and the text widget does not currently support the markdown for strikethrough, which is: ~~strikethrough~~

### Method of discovery
Consultation with the Stanford Office of Digital Accessibility. 

See example:

- See this exhibit feature page: https://exhibits.stanford.edu/sgs-collection/feature/early-students
- Refer to the image in the top row, all the way to the right (has caption: H.S. Gay, Horace W. Joss)
- Zoom in on the image to see the editorial strikethroughs. 
- The strikethroughs should be represented as such in the transcript which can be found in the exhibit appendix: https://exhibits.stanford.edu/sgs-collection/feature/appendix. See the transcript under the heading: "Notes inside page 72 and 73 of John C. Branner's notebook." Please note the strikethroughs are currently being ignored in the transcript because strikethrough markdown is not currently supported in the text widget.

### Suggested solution

We need markdown syntax for strikethroughs to be supported in the Spotlight text widget.
